app/src/main/java/fr/free/nrw/commons/nearby/NearbyController.java

```diff
@@ -98,21 +100,24 @@ public int compare(Place lhs, Place rhs) {
 
         placeList = placeList.subList(0, Math.min(placeList.size(), MAX_RESULTS));
 
+        Bitmap icon = UiUtils.getBitmap(
+                VectorDrawableCompat.create(
```

Comment:
This line appears to cause a connected test error when called by NearbyControllerTest:
```
android.content.res.Resources$NotFoundException: Resource ID #0x7f02005d
at android.content.res.ResourcesImpl.getValue(ResourcesImpl.java:190)
at android.content.res.Resources.getDrawable(Resources.java:766)
at android.support.v4.content.res.ResourcesCompatApi21.getDrawable(ResourcesCompatApi21.java:31)
at android.support.v4.content.res.ResourcesCompat.getDrawable(ResourcesCompat.java:60)
at android.support.graphics.drawable.VectorDrawableCompat.create(VectorDrawableCompat.java:579)
at fr.free.nrw.commons.nearby.NearbyController.loadAttractionsFromLocationToBaseMarkerOptions(NearbyController.java:104)
at fr.free.nrw.commons.NearbyControllerTest.testEmptyList(NearbyControllerTest.java:49)
```